requirements/test.txt

```diff
@@ -12,14 +12,20 @@ defusedxml==0.5.0         # via python3-openid, social-auth-core
 dj-database-url==0.4.2
 django-braces==1.11.0
 django-guardian==1.4.9
-django==1.11.20           # via model-mommy
```

Comment:
we've lost the "via model-mommy" comment. I assume thats irrelevant

Reply:
Yeah, that's an artifact of an older pip-compile run I think. It's a direct dependency, and should always have been!